docs/manage/config-policies.rst

```diff
@@ -0,0 +1,263 @@
+.. _config-policies:
+
+#######################
+ Config Policies Guide
+#######################
+
+Config Policies allow administrators to set limits on how users can define workloads (e.g.,
+experiments, notebooks, tensorboards, shells, and commands). Administrators can include any
+parameter referenced in the :ref:`experiment configuration reference <experiment-config-reference>`.
+
+.. include:: ../_shared/attn-enterprise-edition.txt
+
+Config Policies allow admins to define two types of configurations:
+
+-  **Invariant Configs**: Settings applied to all workloads within a scope (global or workspace).
+-  **Constraints**: Restrictions that prevent users from exceeding resource limits.
+
+These policies are essential for managing task priorities, resource allocation, and setting specific
+configurations across the organization. Global policies override workspace policies, and both
+override user-submitted configurations.
+
+**************
+ Key Features
+**************
+
+-  Apply policies via WebUI or CLI
+-  Define limits for resource usage, environment settings, and more
+-  Apply policies at different levels (cluster-wide or workspace)
+-  Override capabilities for specific user roles or groups
```

Comment:
Hmm, I'm not sure this is a "key feature" of the proj, aside from granting admins the ability to set config policies on a given scope, I don't think the policies themselves provide additional ability to perform user role or group overrides.

Reply:
removed